packages/core-web/src/styles/markbind.css

```diff
@@ -404,23 +404,26 @@ pre {
     position: relative;
 }
 
+/* Override user-agent stylesheets */
+#copy-btn--ua {
```

Comment:
hmm what's the purpose of the separate `id` styling?

would adding it to the existing `copy-btn` class work?

Reply:
I tried that initially (adding to the existing `copy-btn` class), but noticed that it wasn't working on firefox. I thought it was happening because:

Referring to https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

1. User agent stylesheets were being applied *after* markbind's stylesheets
2. User agent stylesheets were being applied on `button` and `[type='button']`
3. For css, if I recall correctly, for the same specificity, the later styles take precendence over earlier defined styles from MDN Specificity), so user agent stylesheets were overriding markbind styles
4. So I used an id selector to override those styles

EDIT: Noticed that button.copy-btn has a higher specificity than .copy-btn alone: https://specifishity.com/;that would be a better solution

Reply:
user agent stylesheets should always have the lowest priority https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade
I tried the `.copy-btn` class alone on firefox, it seems to work. Could it be the work of an extension?


I'm ok with `button.copy-btn` though, which should be better than a generic sounding `copy-btn` class

Reply:
Hmm I tried it again and `.copy-btn` did work. Maybe firefox was sending me cached pages when I tried it out myself... 

I think I'll opt to keep it as `button.copy-btn`, since it would provide more context on what `.copy-btn` should be used in

Reply:
okay